packages/api/src/functions/secureHandler.ts

```diff
@@ -0,0 +1,214 @@
+/*
+
+Based on Stripe's secure webhook implementation
+See: https://stripe.com/docs/webhooks/signatures
+
+*/
+
+import { createHmac } from 'crypto'
+
+import { ForbiddenError } from 'apollo-server-lambda'
+import type { APIGatewayProxyEvent } from 'aws-lambda'
+
+/**
+ * @const {string}
+ */
+const ERROR_MESSAGE = "You don't have access to invoke this function."
+
+/**
+ * @const {number}
+ */
+const FIVE_MINUTES = 5 * 60 * 1000
+
+/**
+ * @const {number}
+ */
+const DEFAULT_TOLERANCE = FIVE_MINUTES
+
+/**
+ * @const {string}
+ */
+const WEBHOOK_SECRET = process.env['WEBHOOK_SECRET'] || ''
```

Comment:
I introduce the ["Nullish Coalescing"](https://www.joshwcomeau.com/operator-lookup?match=nullish-coalescing) operator.
```suggestion
const WEBHOOK_SECRET = process.env.WEBHOOK_SECRET ?? ''
```

Should we throw if this is empty? I presume a person would not want this to be blank.

Reply:
Ah, I see we do throw a bit later down, maybe we don't need the `??` operator at all. We can just presume it has a value, until later when we check it.

Reply:
I had to add the ?? or || '' so that `WEBHOOK_SECRET` would *always* be a string.

Otherwise, I got warnings that setting secret to WEBHOOK_SECRET could be `string | null` and that conflicted with the type. And I don't want secret ever to be null.